--- rgbds/test/CMakeLists.txt	2022-12-04 14:41:03.857199365 -0500
+++ rgbds/test/CMakeLists.txt.orig	2022-12-04 14:40:59.927199129 -0500
@@ -3,7 +3,7 @@
 
 add_executable(rgbgfx_test gfx/rgbgfx_test.cpp)
 
-install(TARGETS randtilegen rgbgfx_test DESTINATION ${rgbds_SOURCE_DIR}/test/gfx)
+# install(TARGETS randtilegen rgbgfx_test DESTINATION ${rgbds_SOURCE_DIR}/test/gfx)
 
 foreach(TARGET randtilegen rgbgfx_test)
   if(LIBPNG_FOUND) # pkg-config
